plugins/source/aws/docs/configuration.md

```diff
@@ -44,7 +67,16 @@ This is used to specify one or more accounts to extract information from.
 
   If specified will use this when assuming role to `role_arn`
 
-## org
+- `default_region` (string) (default: `us-east-1`)
+
+  If specified, this region will be used as the default region for the account.
+
+- `regions` (string)
+
+  Regions to use for this account. Defaults to global `regions` setting.
+
+
+## Organization
```

Comment:
The field name is `org` shouldn't the title of the section match the field name?

Reply:
@bbernays Yeah maybe that would be clearer, but it wasn't the original intention. It's documented as `org` on Line 38. This `Organization` section describes the Organization object, similar to how the setting is `accounts` but the heading is `Account` (singular, describing one Account object)